### PR TITLE
Comment for Migration guide

### PR DESCRIPTION
**Pull request by sidfeiner**
Migrating from Spray to Akka-http, the thing that is most missing in my opinion is the ability to complete a request outside of the router. 
This guide (https://markatta.com/codemonkey/blog/2016/08/03/actor-per-request-with-akka-http/) helped me out a lot - especially the last section "Complete that context". As you've told me, this is intentional. But it still needs to be explicitly explained in the migration guide.

Some more documentation that was missing for me was a clear explanation about the differences between the bindAndHandleXXX versions.

Thank you very very much!

### CODE REVIEW

**Comment by typesafehub-validator** [comment]
Hi @sidfeiner,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

File: docs/src/main/paradox/scala/http/migration-guide/migration-from-spray.md [hunk 1]
```diff
@@ -103,7 +103,7 @@ ToResponseMarshaller.oneOf(
 }
 ```
 
-After:
+After:
```
Comment (ktoso) [inline docs/src/main/paradox/scala/http/migration-guide/migration-from-spray.md]: hmm, sure this is the PR you intended to send? Just a whitespace change... :)

Comment (sidfeiner) [inline docs/src/main/paradox/scala/http/migration-guide/migration-from-spray.md]: I wrote my thoughts as a comment. But I had to edit the file in order to write a comment

Comment (ktoso) [inline docs/src/main/paradox/scala/http/migration-guide/migration-from-spray.md]: Uhm, ok... I see.
I mean, with Pull Requests in open source projects in general a PR is usually for adding the missing/new content, not just commenting what needs to be done. After all, we're an open source project and rely on the communities participation in its development :-)

If you don't want to contribute this section that's too bad, I hoped you'd help us after we helped you on the mailing list... :-) In any case, shall I close this PR or are you considering writing something up still?

Thanks in any case and happy hakking 🎄

**Comment by ktoso** [comment]
Noticed the other PR, awesome thanks :-)

https://github.com/akka/akka-http/pull/680/files

**Pull request closed without merging**